src/components/Map/TopMenu/HamburgerIconButton.tsx

```diff
@@ -22,6 +22,6 @@ export const HamburgerIconButton = ({ anchorRef, onClick }) => (
     color="secondary"
     onClick={onClick}
   >
-    <MenuIcon />
+    <ExpandMoreIcon />
```

Comment:
The chevron is not very visible, can you please leave Hamburger icon, until it is moved to the searchbox?

<img width="532" alt="image" src="https://github.com/zbycz/osmapp/assets/385047/94288355-94cd-4160-976c-2cbd43baa6fe">